docs/user/alerting/create-and-manage-rules.asciidoc

```diff
@@ -22,233 +22,91 @@ available, go to <<alerting-getting-started>>.
-of 5 minutes, you reduce noise by getting emails only every 5 minutes for
-servers that continue to exceed the threshold:
-
-* Minute 1: server X123 > 0.9. _One email_ is sent for server X123.
-* Minute 2: X123 and Y456 > 0.9. _One email_ is sent for Y456.
-* Minute 3: X123, Y456, Z789 > 0.9. _One email_ is sent for Z789.
-
-To get notified only once when a server exceeds the threshold, you can set the
-rule notify setting to `On status changes`. 
-==============================================
---
-
-[role="screenshot"]
-image::images/rule-flyout-general-details.png[alt='All rules have name, tags, check every, and notify properties in common']
-
 [float]
 [[defining-rules-type-conditions]]
 ==== Rule type and conditions
 
-Depending upon the {kib} app and context, you might be prompted to choose the type of rule to create. Some apps will preselect the type of rule for you.
+Depending on the {kib} app and context, you might be prompted to choose the type of rule to create. Some apps will preselect the type of rule for you.
+
+Each rule type provides its own way of defining the conditions to detect, but an expression formed by a series of clauses is a common pattern. For example, in an index threshold rule, the `WHEN` clause enables you to select an aggregation operation to apply to a numeric field.
 
 [role="screenshot"]
-image::images/rule-flyout-rule-type-selection.png[Choosing the type of rule to create]
+image::images/rule-flyout-rule-conditions.png[UI for defining rule conditions on an index threshold rule,500]
 
-Each rule type provides its own way of defining the conditions to detect, but an expression formed by a series of clauses is a common pattern. Each clause has a UI control that allows you to define the clause. For example, in an index threshold rule, the `WHEN` clause allows you to select an aggregation operation to apply to a numeric field.
+All rules must have a check interval, which defines how often to evaluate the rule conditions. Checks are queued; they run as close to the defined value as capacity allows.
 
-[role="screenshot"]
-image::images/rule-flyout-rule-conditions.png[UI for defining rule conditions on an index threshold rule]
+For details on what types of rules are available and how to configure them, refer to <<rule-types>>.
 
 [float]
 [[defining-rules-actions-details]]
-==== Action type and details
-
-Actions are optional when you create a rule. However, to receive notifications when a rule meets the defined conditions, you must add one or more actions. Start by selecting a type of connector for your action:
-
-[role="screenshot"]
-image::images/rule-flyout-connector-type-selection.png[UI for selecting an action type]
+==== Actions
 
-Each action must specify a <<alerting-concepts-connectors, connector>> instance. If no connectors exist for the selected type, click **Add connector** to create one.
+You can add one or more actions to your rule to generate notifications when its
```

Comment:
We also give the option to not add any actions.

Reply:
Is there a particular use case where we don't expect rules to have actions? If yes, I agree that's likely worth calling out.